faq/index.md

```diff
@@ -13,6 +13,13 @@ Yes, SysKit Security Manager works with both SharePoint Online and On-Premise si
 If you are a __Global Administrator__ in an Office 365 tenant or a __Farm Administrator__ in a SharePoint On-Premise environment, you will be able to automatically discover and connect to all site collections (including Office 365 Groups) in the tenant or farm.
 If you are a __Site Collection Administrator__, you will be able to add multiple connections manually by entering the site collection URLs or by importing a text file with the site collection URLs, newline separated.
 
+## Can I see OneDrive reports without making myself an administrator to user OneDrive accounts?
+If you are a Global Office 365 tenant administrator, you’ll have an option in the connection wizard to [connect to all OneDrive accounts](#internal/how-to/connect-to-office-365).
+
+The only way we can access user data and create OneDrive reports is if admin is site collection administrator to user OneDrive accounts. Unfortunately , there is currently no other supported way by Microsoft to do so. Please note that these reports are optional, and if you do not wish to explore them just skip the step in the Connection Wizard. No permissions will be granted to you when it comes to OneDrive sites.
```

Comment:
to user OneDrive accounts => on user OneDrive account :)

Reply:
fixano